lib/util.ts

```diff
@@ -1,16 +1,21 @@
-'use strict';
-const fs = require('fs');
-const thrift = require('thrift');
-const parquet_thrift = require('../gen-nodejs/parquet_types')
-
+import thrift from "thrift";
```

Comment:
After converting from require to import. A lot of the functions from thrift are shown as not existing when they do exist within the node modules